Static_agent/Slither_agent/slither_to_json_with_diagnostics.py

```
@@ -272,3 +272,89 @@ def parse_slither_cli_output(stdout, stderr, contract_path):
+
+            for r in results:
+                try:
+                    check = getattr(r, "check", None) or getattr(r, "name", None) or (r.get("check") if isinstance(r, dict) else None)
+                    impact = getattr(r, "impact", None) or getattr(r, "severity", None) or (r.get("impact") if isinstance(r, dict) else None) or "Unknown"
+                    desc = getattr(r, "description", None) or (r.get("description") if isinstance(r, dict) else None) or ""
+                    elems = getattr(r, "elements", None) or (r.get("elements") if isinstance(r, dict) else None)
+                    if elems is None:
+                        nodes = getattr(r, "nodes", None) or (r.get("nodes") if isinstance(r, dict) else None)
+                        elems = [str(n) for n in nodes] if nodes else [str(r)]
+                except Exception:
+                    check = str(r); impact = "Unknown"; desc = ""; elems = [str(r)]
```

Ruff (0.13.1)

Multiple statements on one line (semicolon)

(E702)